dev/com.ibm.websphere.javaee.annotation.1.3/bnd.bnd

```diff
@@ -0,0 +1,29 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.websphere.javaee.annotation.1.3; singleton:=true
+
```

Comment:
Shouldn't this also say...?
`Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"`